master/getting-started/kubernetes/tutorials/app-layer-policy/index.md

```diff
@@ -0,0 +1,298 @@
+> [view the manifest in your browser](manifests/30-policy.yaml){:target="_blank"}.
+{: .alert .alert-info}
+
+Let's examine this policy piece by piece.  It consists of 3 policy objects, one for each microservice.
+
+    apiVersion: projectcalico.org/v3
+    kind: GlobalNetworkPolicy
+    metadata:
+      name: customer
+    spec:
+      selector: app == 'customer'
+      ingress:
+        - action: Allow
+          http:
+            methods: ["GET"]
+      egress:
+        - action: Allow
+
+This policy protects the customer web app.  Since this application is customer facing, we do not
+restrict what can communicate with it.  We do, however, restrict that only HTTP `GET` requests are
+allowed.
+
+    apiVersion: projectcalico.org/v3
+    kind: GlobalNetworkPolicy
+    metadata:
+      name: summary
+    spec:
+      selector: app == 'summary'
+      ingress:
+        - action: Allow
+          source:
+            serviceAccounts:
+              names: ["customer"]
+      egress:
+        - action: Allow
+
+The second policy protects the account summary microservice.  We know the only consumer of this
+service is the customer web app, so we restrict the source of incoming connections to the service
+account for the customer web app.
+
+    apiVersion: projectcalico.org/v3
+    kind: GlobalNetworkPolicy
+    metadata:
+      name: database
+    spec:
+      selector: app == 'database'
+      ingress:
+        - action: Allow
+          source:
+            serviceAccounts:
+              names: ["summary"]
+      egress:
+        - action: Allow
+
+The third policy protects the database.  Only the summary microservice should have direct access to
+the database.
+
+Let's verify our policy is working as intended.  First, return to your browser and refresh, to
+ensure policy enforcement has not broken the application.
+
+Next, return to the customer web app.  Recall that we simulated an attacker gaining control of that
+pod by `exec`ing into it.
+
+    kubectl exec -ti customer-<fill in pod ID> -c customer bash
+
+Repeat our attempt to access the database.
+
+    curl http://database:2379/v2/keys?recursive=true
+
+This time we should get a 403 Forbidden response (we have left out the JSON formatting because we do
+not expect to get a valid JSON response).  Only the account summary microservice has database access
+according to our policy.
+
+Finally, let's return to the attack pod that simulated stealing secret keys.
+
+    kubectl exec -ti attack-<fill in pod ID> bash
+
+Let's repeat our attack with stolen keys (we'll further increase the account balance to highlight
+whether it succeeds).
+
+    curl -k https://database:2379/v2/keys/account/519940/balance -d value="99999.99" -XPUT --key /etc/certs/key.pem --cert /etc/certs/cert-chain.pem
+    
+You should get no response, and refreshing your browser should not show an increased balance.
+
+You might wonder how Calico was able to detect and prevent this attack---the attacker was able to
```

Comment:
s/Calico/`{{site.prodname}}` (both instances)

Also, the triple hyphens does not render as an em dash in our site. To get an em dash, you need to paste in the actual em dash character.